samples/drivers/led_lp50xx/src/main.c

```diff
@@ -202,6 +203,7 @@ static int run_channel_test(const struct device *lp50xx_dev,
 		k_sleep(SLEEP_DELAY);
 
 		/* Turn LEDs on. */
+		memset(buffer, 0, sizeof(buffer));
```

Comment:
Is this `memset` (and all the others after) needed ?

Reply:
I added them because I had some glitches on my board due to leftover values between color / brightness channel settings. Maybe not all are required but it creates some sort of pattern.

Do you prefer me to keep the strict minimum ?